DataFormats/Reconstruction/include/ReconstructionDataFormats/TrackFwd.h

```diff
@@ -96,6 +119,7 @@ class TrackParFwd
   void propagateParamToZlinear(double zEnd);
   void propagateParamToZquadratic(double zEnd, double zField);
   void propagateParamToZhelix(double zEnd, double zField);
+  void getCircleParams(float bz, o2::math_utils::CircleXY<float>& c, float& sna, float& csa) const;
```

Comment:
A comment for this method would be welcome : what is this "circle" ? (and looking at the implementation I see two cases, maybe worth nothing them briefly in the comment if that's relevant) ?